src/components/Gradebook/index.jsx

```diff
@@ -260,6 +283,16 @@ export default class Gradebook extends React.Component {
                     </label>
                   </span>
                 </div>
+                { this.props.assignmnetTypes.length > 0 &&
```

Comment:
With this drop down, can you please remove the `category` radio button elements on top?

Reply:
yes! done!